src/components/client-factory.ts

```diff
@@ -116,6 +119,34 @@ export class ClientFactory {
                 callback(err, response, body);
             });
         });
+
+        // matrix-js-sdk uses the `loglevel` logging library for it's logging
+        // but the only way to get it to log to winston is to modify the
+        // global methodFactory.
+        // To avoid potentially breaking other bridges, this is off by default.
+        loglevel.methodFactory = (methodName, _logLevel, loggerName) => {
+            return (...args) => {
+                const loggerInstance = logging.get(`js-sdk:${loggerName}`);
+                if (methodName === "info" || methodName === "debug" ||
```

Comment:
Real nit here, but I'd remove `methodName === "info" ||` from this line, let it fall through to the else, feels more semantically correct.